### ISSUE TITLE
Questions Regarding android.widget.FrameLayout

### ISSUE THREAD

**Issue by daphneyyt** [microsoft/accessibility-insights-for-android-service#193]
Hi Accessibility Insights for Android team, we have 2 questions regarding a class name that was flagged in our WCAG checks.

2) May we check what does Class Name **android.widget.FrameLayout** mean? 
2a) what is a definition of an android.widget.FrameLayout

2b where is the specific location on the mobile app is AIFA highlighting?

2c) what should we do to rectify this issue?

Please find the corresponding screenshots of this encounter:
![WCAG frame layout 1](https://user-images.githubusercontent.com/75086091/210995809-13cd3e50-7c34-4b27-9ff1-5357f98ff644.png)

![WCAG frame layout 2](https://user-images.githubusercontent.com/75086091/210995850-338603fd-f309-45b8-bb6d-6147a3f5afb8.png)

As this mobile app requires a login, we are unable to share these details with the AFIA team. However, we would appreciate it if you could provide us with some insights into this.

If the team needs more details, we will be happy to schedule a zoom call to show the team the issue encountered.

**Comment by pownkel** [comment]
Hi, thanks for using Accessibility Insights for Android! Regarding your questions:

2a) The android.widget.FrameLayout class means this is a [FrameLayout ](https://developer.android.com/reference/android/widget/FrameLayout)object

2b) I'd probably need to inspect the app to answer this question. If you're developing with Android Studio, you can use the [Layout Inspector](https://developer.android.com/studio/debug/layout-inspector) to look through the view hierarchy and find any unlabeled FrameLayouts in the app. If there is no FrameLayout in that location, this is a bug in our service that we'll need to investigate.

2c) See our [docs on the ActiveViewName rule](https://accessibilityinsights.io/info-examples/android/active-view-name/) (also accessible in the app through the "More information about ActiveViewName" link). In this case, the FrameLayout is active and therefore able to be focused, but has no text available to the screen reader. You can fix this by adding an appropriate contentDescription or text label to the FrameLayout.

I hope that helps, please let us know if you need further clarification!

**Comment by daphneyyt** [comment]
Dear @pownkel thank you for your speedy response and for your explanation on android.widget.FrameLayout

The links provided are most helpful.
